test/e2e/helpers.js

```diff
@@ -46,6 +48,15 @@ async function withFixtures(options, testSuite) {
   let failed = false;
   try {
     await ganacheServer.start(ganacheOptions);
+    let contractRegistry;
+
+    if (smartContract) {
+      const debugGanacheSeeder = true;
+      const ganacheSeeder = new GanacheSeeder(debugGanacheSeeder);
```

Comment:
Nit: Can we avoid the `debugGanacheSeeder` var and simply pass `true` here?

Reply:
I think the intention of this approach is for clarity reasons: so code is more understandable, as we explicitly say we are activating the debugger. But surely we can pass true directly, and avoid one extra line. (this part of code was initially done by @david0xd). Ty!